incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,19 @@
+//////////  S V G   M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
```

Comment:
Can we add some example usage here? We don't do this currently but I think we should be providing much better documentation on how and where to use all the mixins/helpers in this project.

Perhaps some examples of how to call it with the different options?